Whiskey/Tasks/Invoke-WhiskeyNspCheck.ps1

```diff
@@ -91,9 +101,19 @@ function Invoke-WhiskeyNspCheck
     }
 
     Write-Timing -Message 'Installing NSP'
-    $nspModuleRoot = Install-WhiskeyNodeModule -Name 'nsp' -Version '2.7.0' -ApplicationRoot $workingDirectory -RegistryUri $npmRegistryUri -ForDeveloper:$TaskContext.ByDeveloper
+
+    if( $TaskParameter['NspVersion'] )

```

Comment:
I think this property should actually be named just `Version` without the `Nsp` to be consistent with the other tasks that have a `Version` property.

Reply:
Agreed, I've updated to match the standard convention!